README.md

```diff
@@ -42,29 +50,53 @@ in another terminal if you are doing front end work:
 
 
 - store environment variables
- cf cups VCAP_SERVICES -p "{'SECRET_KEY': 'replace-with-your-secret-key'}"
+ cf cups VCAP_SERVICES -p SECRET_KEY
+
+when prompted give it the secret key
+
+
+You will needed to enable CORS via awscli, instructions are here: https://cloud.gov/docs/services/s3/#allowing-client-side-web-access-from-external-applications
+
+
+Right now, the route is set for the production space, we will want to pass in different routes for different spaces but that can be handled when we add the automation.
+
+To deploy run:
 
-(I had to give it the varable name in this command and follow up with the secret key, I am not sure why)
+    cf push
+
+That will push to cloud.gov according to the instructions in the manifest and Profile.
+
+### Create admin accounts
 
 Need to ssh to create superuser (would like to do this automatically in another PR)
 
     cf ssh crt-portal-django
```

Comment:
I haven't set up CloudFoundry locally yet, would like to go through some of these steps as part of review.

Reply:
Configured locally during pairing time today!